spec/support/macros.rb

```diff
@@ -0,0 +1,12 @@
+def set_current_user
```

Comment:
This method should accept a user and create a new one as default (if no user was passed). You are going to need that later on.